packages/features/form-builder/schema.ts

```diff
@@ -28,6 +28,7 @@ export const EditableSchema = z.enum([
   "system-but-hidden", // Can't be deleted, name can't be edited, will be shown
   "user", // Fully editable
   "user-readonly", // All fields are readOnly.
+  "user-owner-secret", // Only shared between owner and attendee, no one else can see this response
```

Comment:
The idea of EditableSchema was that it talks about different editability scenarios of a field. It doesn't make sense to have a `user-owner-secret` here.